### PR TITLE
Fixed typos in README. Used markdown just to be more Go-like.

### PR DESCRIPTION
**Pull request by normanjaeckel**
Please read carefully. I fixed a lot of typos but also made small changes in the sentences (hopefully without changing the meaning).

### CODE REVIEW

**Comment by ostcar** [comment]
I added some commit, please look at them.

Also: Why do you think, that markdown is more go-like?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+geiss
```
Comment (normanjaeckel) [inline README.md]: Here (and on some other places below) we should also write `Geiss` if the name of the project starts with a capital letter. I understood your change in line 4 that you want to name the project `Geiss` and not `geiss`. If you want your project to be named `geiss`, I am not sure whether line 4 and 42 have to be changed. We have the proper name (like a trademark) on the one hand and the English rule that sentences start with a capital letter on the other.

???

**Review by normanjaeckel — COMMENTED** [review]
One comment.

I think Markdown is more Go like because many Go projects like Go itself and Gorilla use it. Also it is sometimes better readable than RST which is a little bit old fashioned. Parsing MD is easier (you do not need docutils with compiled dependencies), my Geany has a MD preview, ...

But of course you can also see it as a personal question of taste.

:-)

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Oskar Hahn
+Copyright (c) since 2016 Oskar Hahn
```
Comment (normanjaeckel) [inline LICENSE]: I think the year in this file marks the year of the constitution of the copyright for this intelectual property. This is relevant for the year when the copyright expires. So you do not have to change the file every year but only if you add some changes in a new year.

What about adding a small script to Travis CI that checks every pull request and checks if the current year is present?

**Comment by ostcar** [comment]
ok. Geiss the name of the software has now a capital G. ```geiss``` the name of the executable, does not. ```geiss``` the repo path does not neither.

I orientated me on other projects like atom.io and django.

Anything else?

**Comment by normanjaeckel** [comment]
Looks good.

**Pull request merged**